update readme (#985)
SpikeInterface/spikeinterface: README.md

- post-process sorted datasets.
- compare and benchmark spike sorting outputs.
- compute quality metrics to validate and curate spike sorting outputs.

> - visualize recordings and spike sorting outputs in sevral ways (matplotlib, sortingview, in jupyter)

Reviewer: ```suggestion
- visualize recordings and spike sorting outputs in several ways (matplotlib, sortingview, in jupyter)
```

> - visualize recordings and spike sorting outputs with sevral ways (matplotlib, sorting view, in jupyter)

Reviewer: ```suggestion
- visualize recordings and spike sorting outputs in sevral ways (matplotlib, sortingview, in jupyter)
```

## Documentation

All documentation for spikeinterface can be found [here](https://spikeinterface.readthedocs.io/en/latest).

> Some usefull tutorials in jupyter notebook format [here](https://github.com/SpikeInterface/spiketutorials).

Reviewer: ```suggestion
Some useful jupyter notebook tutorials can be found in [spiketutorials](https://github.com/SpikeInterface/spiketutorials).
```

> There are also some usefull notebooks [on our blog](https://spikeinterface.github.io) for advanced benchmarking or sorting components.

Reviewer: ```suggestion
There are also some useful notebooks [on our blog](https://spikeinterface.github.io) that cover advanced benchmarking and sorting components.
```
